scripts/testing/setup-mock-test-env.py

```diff
@@ -197,19 +210,39 @@ def check_args(namespace):
 
 def get_required_packages():
     """Get required packages for running Anaconda tests."""
-    script = _get_script_dir() + os.path.sep + DEPENDENCY_SOLVER
+    script = _get_dependency_script_path()
     cmd = [script]
 
     proc_res = _check_subprocess(cmd, "Can't call dependency_solver script.", stdout_pipe=True)
 
     return proc_res.stdout.decode('utf-8').strip()
 
 
+def get_required_pip_packages():
+    """Get pip packages for running Anaconda tests."""
+    script = _get_dependency_script_path()
+    cmd = [script, "--pip"]
+
+    proc_res = _check_subprocess(cmd, "Can't call dependency_solver script.", stdout_pipe=True)
+
+    return proc_res.stdout.decode('utf-8').strip()
+
+
 def install_required_packages(mock_command):
     packages = get_required_packages()
     install_packages_to_mock(mock_command, packages)
 
 
+def install_required_pip_packages(mock_command):
+    packages = get_required_pip_packages()
+    install_pip_packages(mock_command, packages)
+
+
+def install_pip_packages(mock_command, addon_packages):
```

Comment:
It looks like this method could be removed and replaced with `install_pip_packages_to_mock`, because it doesn't do anything useful.

Reply:
Yes it doesn't but it is consistent. The *_mock methods are something like private. They are working in mock directly and they shouldn't be used in the main.
I want to have this structure. Ideally it should be placed to multiple files but that would complicate things...